hack/istio/install-testing-demos.sh

```diff
@@ -10,15 +10,23 @@
 set -eu
   
 install_sleep_app() {
+
+  if [[ "${ISTIO_DIR}" = "" ]]; then
```

Comment:
this is trivial, but we should always use the same kind of syntax when possible. This is using what I think is some bash specific syntax (the double brackets) but everywhere else we are using single-bracket notation with the double-equals.

```suggestion
  if [ "${ISTIO_DIR}" == "" ]; then
```